include/opendht/dht_proxy_server.h

```diff
@@ -313,6 +384,15 @@ class OPENDHT_PUBLIC DhtProxyServer
     std::map<std::string, PushListener> pushListeners_;
     proxy::ListenToken tokenPushNotif_ {0};
 #endif //OPENDHT_PUSH_NOTIFICATIONS
+
+    const std::string RESP_MSG_DESTINATION_NOT_FOUND = "{\"err\":\"No destination found\"}";
```

Comment:
Note: I think this can be replaced by constexpr char*

Reply:
Also only in .cpp if constexpr

Reply:
Done at bc2737c